### PR TITLE
Add key with sequence to get correct timeseries for a grid action

### PR DESCRIPTION
**Pull request by vanmeeuwen**

### CODE REVIEW

File: src/lib/ChartConfig/timeSeriesDisplayToChartConfig.ts [hunk 1]
```diff
@@ -1,13 +1,14 @@
...
 import { ChartSeries } from "@/components/TimeSeriesComponent/lib/ChartSeries.js"
-import { TimeSeriesDisplaySubplot, TimeSeriesDisplaySubplotItem } from "@deltares/fews-pi-requests"
 import { cssStyleFromFewsLine, cssStyleFromFewsMarker, chartMarkerFromFews } from "./Styles"
+import { TimeSeriesDisplaySubplot, TimeSeriesDisplaySubplotItem } from "../TimeSeries/types"
 
 export function timeSeriesDisplayToChartConfig(subplot: TimeSeriesDisplaySubplot, title: string): ChartConfig {
   const config: ChartConfig = {
     title: title,
-    xAxis: [],
+    sequence: subplot.items[0].sequence !== undefined ? subplot.items[0].sequence : undefined,
```
Comment (amarkensteijn) [inline src/lib/ChartConfig/timeSeriesDisplayToChartConfig.ts:10]: Assumes there is never more than one item per subplot (which might be true for this use case)

File: src/lib/TimeSeries/types.ts [hunk 2]
```diff
@@ -1,3 +1,6 @@
+import { TimeSeriesDisplaySubplotItem as SubPlotItem,
+  TimeSeriesDisplaySubplotItemAxis } from "@deltares/fews-pi-requests";
```
Comment (amarkensteijn) [inline src/lib/TimeSeries/types.ts:2]: Add type to import:
```suggestion
import type { TimeSeriesDisplaySubplotItem as SubPlotItem,
  TimeSeriesDisplaySubplotItemAxis } from "@deltares/fews-pi-requests";

```

File: src/lib/TimeSeries/types.ts [hunk 3]
```diff
@@ -24,3 +27,13 @@ export enum SeriesOperationType {
...
+
+export interface TimeSeriesDisplaySubplotItem extends SubPlotItem{
+  sequence?: number;
+}
+
+
+export interface TimeSeriesDisplaySubplot {
+  xAxis?: TimeSeriesDisplaySubplotItemAxis;
+  items: TimeSeriesDisplaySubplotItem[];
+}
```
Comment (amarkensteijn) [inline src/lib/TimeSeries/types.ts:39]: Should both be updated in the `fews-pi-request` library.

Comment (vanmeeuwen) [reply src/lib/TimeSeries/types.ts:40]: is requested in https://issuetracker.deltares.nl/browse/FEWS-29590 and https://github.com/Deltares/fews-wms-requests/issues/21

File: src/lib/ChartConfig/timeSeriesDisplayToChartConfig.ts (hunk 1 above)
Comment (vanmeeuwen) [reply src/lib/ChartConfig/timeSeriesDisplayToChartConfig.ts:10]: will be retrieved from the backend soon

**Pull request merged**